docs/how_tos/5_how_to_run_circuits_on_Braket_local_backend.ipynb

```diff
@@ -0,0 +1,128 @@
+{
```

Comment:
Would it make sense to merge with [this](https://github.com/dakk/qiskit-braket-provider/blob/get_statevector/docs/how_tos/2_how_to_retrieve_results_from_backend.ipynb) how-to, since the primary motivation is to show how to retrieve a specific result (state vector) here?